### DISCUSSION TITLE
IPA 资源库中，来源地址应该填什么格式

### DISCUSSION THREAD

**Discussion by seepine** [PlayCover/PlayCover#1151] [Q&A] [answered]
<img width="787" alt="image" src="https://github.com/PlayCover/PlayCover/assets/93666053/56b2c0e2-9bdb-4e71-a088-f25f0bfdeffd">

**Comment by TheMoonThatRises** [comment, marked as answer]
Take a look at the documentation here: https://docs.playcover.io/other_features/ipa_library.
